mc/gui/main_win.py

```diff
@@ -72,43 +91,49 @@ def __init__(self):
         self.out_text_qll.setWordWrap(True)
-        self.breathing_history_wt = mc.gui.breathing_history_wt.BreathingHistoryWt()
-        vbox_l4.addWidget(self.breathing_history_wt)
 
-        vbox_l4 = QtWidgets.QVBoxLayout()
-        hbox_l3.addLayout(vbox_l4)
-        self.br_phrase_list_wt = mc.gui.breathing_phrase_list_wt.BreathingPhraseListWt()
+    def add_breathing_phrase_list_to_main_container(self, hbox_l3):
+        vbox_l4 = self.new_panel_in_main_window(hbox_l3)
         vbox_l4.addWidget(self.br_phrase_list_wt)
         self.br_phrase_list_wt.selection_changed_signal.connect(self.on_breathing_list_row_changed)
         self.br_phrase_list_wt.phrase_changed_signal.connect(self.on_breathing_phrase_changed)
-        self.br_settings_wt = mc.gui.breathing_settings_wt.BreathingSettingsWt()
-        vbox_l4.addWidget(self.br_settings_wt)
-        self.br_settings_wt.updated_signal.connect(self.on_breathing_settings_changed)
-        self.br_settings_wt.breathe_now_button_clicked_signal.connect(self.open_breathing_dialog)
+        self.configure_breathing_settings(vbox_l4)
 
-        vbox_l4 = QtWidgets.QVBoxLayout()
-        hbox_l3.addLayout(vbox_l4)
-        self.rest_action_list_wt = mc.gui.rest_action_list_wt.RestActionListWt()
+    def add_rest_action_list_to_main_container(self, hbox_l3):
+        vbox_l4 = self.new_panel_in_main_window(hbox_l3)
         vbox_l4.addWidget(self.rest_action_list_wt)
         self.rest_action_list_wt.update_signal.connect(self.on_rest_action_list_updated)
         self.rest_action_list_wt.selection_changed_signal.connect(self.on_rest_action_list_row_changed)
-        self.rest_settings_wt = mc.gui.rest_settings_wt.RestSettingsWt()
+        self.configure_rest_settings(vbox_l4)
+
+    def new_panel_in_main_window(self, hbox_l3):
+        vbox_l4 = QtWidgets.QVBoxLayout()
+        hbox_l3.addLayout(vbox_l4)
+        return vbox_l4
+
+    def configure_rest_settings(self, vbox_l4):
         vbox_l4.addWidget(self.rest_settings_wt)
         self.rest_settings_wt.settings_updated_signal.connect(self.update_rest_timer)
         self.rest_settings_wt.rest_now_button_clicked_signal.connect(self.on_rest_rest)
         self.rest_settings_wt.rest_reset_button_clicked_signal.connect(self.update_rest_timer)
         self.rest_settings_wt.rest_slider_value_changed_signal.connect(self.on_rest_slider_value_changed)
 
-        # Setup of Menu
-        self.menu_bar = self.menuBar()
-        self.update_menu()
-
-        # Setup of Timers
-        self.on_breathing_settings_changed()
-        self.update_rest_timer()
+    def configure_breathing_settings(self, vbox_l4):
+        vbox_l4.addWidget(self.br_settings_wt)
+        self.br_settings_wt.updated_signal.connect(self.on_breathing_settings_changed)
+        self.br_settings_wt.breathe_now_button_clicked_signal.connect(self.open_breathing_dialog)
 
-        # Setup of Systray
-        self.setup_systray()
+    def set_window_title_with_version_and_storage(self):
```

Comment:
maybe just set_window_title, title may not use version and storage information in the future